src/llamacrawl/services/sessions/document_builder.py

```
@@ -0,0 +1,163 @@
+
+        Example:
+            >>> conv_doc = SessionsDocumentBuilder.build_conversation_document(
+            ...     documents=[msg1, msg2, msg3],
+            ...     file_path=Path("session.jsonl"),
+            ...     source_type="claude_code"
+            ... )
+            >>> conv_doc.doc_id
+            'conv:session-uuid'
+        """
+        if not documents:
+            raise ValueError("Cannot build conversation document from empty list")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)